coordinator/tributary/tendermint/src/lib.rs

```diff
@@ -796,6 +796,9 @@ impl<N: Network + 'static> TendermintMachine<N> {
       if self.block.log.has_consensus(self.block.round().number, Data::Prevote(Some(block.id()))) {
         match self.network.validate(block).await {
           Ok(_) => (),
+          // BlockError::Temporal is due to us not having the locally provided tx yet,
```

Comment:
Tendermint should not acknowledge Tributary.

Though yes, the logic

"If there's a temporal error yet no one else has it, we assume it's our fault"

is perfectly sound :)